frontend/main-app/src/components/Login/LoginUtils.ts

```diff
@@ -0,0 +1,27 @@
+import { z } from "zod";
+import { LoginForm, LoginFormError } from "./types";
+
+type IValidateLoginForm = {
+  loginForm: LoginForm;
+  setFormErrors: (value: LoginFormError | null) => void;
+};
```

Comment:
Should this maybe moved to Login/types.d.ts? Is there a common practice about this in the Studio repo?

Reply:
This is a good question.
I haven't given it much thought, but I opted for using "I" in the type's name as a naming convention, avoiding conflicts with the function name. In the studio, it appears that there are no strict rules regarding this.

I placed this type in the LoginUtils file since it's exclusively used within that file. The other types in types.d.ts are utilized across multiple files. However, I'm open to reconsidering this decision. If you believe it's better to move that type to types.d.ts as well, I'm ok to make that change :) 

Shall we discuss the typing structure further in our next meeting? :)

Reply:
Sounds good 😊